config/samples/cxx_example_loadtest.yaml

```diff
@@ -50,6 +50,21 @@ spec:
   # results:
   #   bigQueryTable: "example-project.foo.demo_dataset"
 
+  # timeoutSeconds field is to advise the longest time a test
+  # could possibly run, test runs longer than given value will be marked as
+  # Error and would not be allocate more resource. The field takes integer
```

Comment:
nit: I think the state is past tense, Errored. I think this explains the details. I think @paulosjca may have feedback on wording.

Reply:
I definitely should change this since it has to be in agreement with states names in Loadtest.LoadTestState.